source/text/position2dlabel.ts

```diff
@@ -1,180 +1,182 @@
-//     }
-
-//     /**
-//      * Sets the 2D position of the label's reference point (i.e. lower left corner for horizontal alignment).
-//      */
-//     set position(xy: vec2) {
-//         this._position = vec2.clone(xy);
-//     }
-//     get position(): vec2 {
-//         return this._position;
-//     }
-
-//     /**
-//      * Sets 2D position parameters as specified in OpenLL. Position is the label's reference point (i.e. lower left
-//      * corner for horizontal alignment).
-//      * @param x - x coordinate of the 2D position
-//      * @param y - y coordinate of the 2D position
-//      * @param unit - the unit to interpret the coordinates
-//      */
-//     setPosition(x: number, y: number, unit?: Label.SpaceUnit): void {
-//         /** @todo assert that SpaceUnit is px or pt; transform to NDC? */
-//         this._position = vec2.fromValues(x, y);
-//     }
-
-//     /**
-//      * Sets the 2D direction of the label, i.e., the direction of the baseline.
-//      */
-//     set direction(xy: vec2) {
-//         vec2.normalize(this._direction, xy);
-//     }
-//     get direction(): vec2 {
-//         return this._direction;
-//     }
-
-//     /**
-//      * Sets the 2D direction parameters as specified in OpenLL. The labels's direction is the direction of its
-// baseline.
-//      * @param x - x coordinate of the 2D direction vector.
-//      * @param y - y coordinate of the 2D direction vector.
-//      */
-//     setDirection(x: number, y: number): void {
-//         this.direction = vec2.fromValues(x, y);
-//     }
-
-//     /**
-//      * This unit is used for the font size. This method overrides the super.fontSizeUnit, since a position2dlabel
-// only
-//      * allows px, not World.
-//      * (@see {@link fontSize})
-//      * @param newUnit - unused, since there is only one allowed unit (Px) for this kind of label
-//      */
-//     set fontSizeUnit(newUnit: Label.SpaceUnit) {
-//         log(LogLevel.Warning, `New SpaceUnit ${newUnit} not set; only allowed SpaceUnit is Px for this label.`);
-//     }
-
-// }
+/**
+ * A Label that can be positioned in 2D space. The unit for positions, size and transformations, is pixel (px).
+ */
+export class Position2DLabel extends Label {
+
+    /** @see {@link position} */
+    protected _position: vec2;
+    /** @see {@link direction} */
+    protected _direction: vec2;
+    /** @see {@link frameSize} */
+    protected _frameSize: vec2;
+
+
+    /**
+     * Constructs a pre-configured 2D-label with given text
+     * @param text - The text that is displayed by this label.
+     * @param fontFace - The font face that should be used for that label, or undefined if set later.
+     */
+    constructor(text: Text, type: Label.Type, fontFace?: FontFace) {
+        super(text, type, fontFace);
+        this._position = vec2.fromValues(0.0, 0.0);
+        this._direction = vec2.fromValues(1.0, 0.0);
+
+        this._frameSize = vec2.create();
+
+        this._fontSize = 20;
```

Comment:
can you please refine the default font sizes for label and position2d und position3d labels as follow:
prefer a protected static readonly DEFAULT_FONT_SIZE in specialized classes, and no font size in Abstract Label (e.g., see Label.DEFAULT_ELLIPSIS)